### PR TITLE
[TE] Show more debug information in preview

### PR DESCRIPTION
**Pull request by xiaohui-sun**
1. Print up to 5 stack messages in preview.
2. Change the fake detection id to 0.

### CODE REVIEW

File: thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java [hunk 1]
```diff
@@ -576,7 +576,7 @@ public Response yamlPreviewApi(
       Map<String, Object> newDetectionConfigMap = new HashMap<>(ConfigUtils.getMap(this.yaml.load(payload)));
       DetectionConfigDTO detectionConfig = buildDetectionConfigFromYaml(tuningStart, tuningEnd, newDetectionConfigMap, null);
       Preconditions.checkNotNull(detectionConfig);
-      detectionConfig.setId(Long.MAX_VALUE);
+      detectionConfig.setId(0L);
```
Comment (jihaozh) [inline thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java]: I remember we had some issues before to set this to 0. Could we keep this unchanged?

Comment (xiaohui-sun) [reply thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java]: Do you know what issue?
Long.MAX_VALUE is very long in alert message. How about setting it to -1?

Comment (xiaohui-sun) [reply thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java]: ok, I reverted first.

Comment (jihaozh) [reply thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java]: Setting to 0 will cause fetching all the anomalies from the database in preview. we can keep it for now.

**Pull request merged**